### PR TITLE
Updated Deep Dive

### PR DESCRIPTION
**Pull request by jason-fox**
Added links to all Developer Track Presentations from Malaga and Porto.

### CODE REVIEW

File: doc/big-data-analysis-of-historic-context-information/deep-dive.md [hunk 1]
```diff
@@ -9,3 +9,11 @@ Should you wish to learn much more about this, please check:
        - [Github repository (reference implementation)](https://github.com/Fiware/context.Cosmos)
        - [Read the full documentation](http://fiware-cosmos.readthedocs.org/en/latest/)
        - [Latest stable release](https://github.com/Fiware/context.Cosmos/releases/latest)
+
+## Presentations
+
+Presentations about Data Analysis from recent FIWARE Global Summits:
+
+| Porto 2018 | Malaga 2017 |
+|------------|-------------|
```
[the remark this answers is not in the document]
Comment (jason-fox) [reply doc/big-data-analysis-of-historic-context-information/deep-dive.md]: NTD - I didn't find a relevant Porto Presentation

File: doc/creating-application-dashboards/deep-dive.md [hunk 2]
```diff
@@ -1,7 +1,16 @@
...
    - [Read the full documentation](http://wirecloud.readthedocs.org/en/latest/)
    - [Latest stable release](https://github.com/Fiware/apps.Wirecloud/releases/latest)
+
+## Presentations
+
+Presentations about Data Visualization from recent FIWARE Global Summits:
+
+| Porto 2018 | Malaga 2017 |
+|------------|-------------|
+|[Business Analytics using Knowage](https://www.slideshare.net/FI-WARE/fiware-global-summit-business-intelligence-using-knowage-97030885)|[Building Wirecloud Real-time Dashboards](https://www.slideshare.net/FI-WARE/fiware-tech-summit-miguel-jimenez-building-realtime-dashboards-to-monitor-context)|
```
[the remark this answers is not in the document]
Comment (jason-fox) [reply doc/creating-application-dashboards/deep-dive.md]: Fixed - 3eca867 - I've amended the Deep-dive topic titles to match Chapter or Sub Chapter.

I've added more Wirecloud and Knowage links and renamed the section title to Visualizations for now. A Knowage intro needs to be added to the texts at some point (which is not in scope for this PR) Similar intros are needed for other incubated enablers, and some of the existing intros need tiding up.

**Comment by jmcanterafonseca** [comment]
@jason-fox please could you resolve the conflicts, so that we can finally merge?

**Comment by jmcanterafonseca** [comment]
as a best practice it is better to rebase and not merge with ...

**Pull request merged**